src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Shared/Styles/XamlDisplay.xaml

```diff
@@ -39,6 +39,31 @@
 		</Setter>
 	</Style>
 
+	<Style x:Key="StretchedXamlDisplayOnTheSideStyle"
```

Comment:
You can instead use a `XamlDisplayBelowStyle` or (`ContentOnlyXamlDisplayStyle` + `smtx:XamlPresenter`)